src/main/java/uk/gov/mca/beacons/service/model/BeaconPerson.java

```diff
@@ -43,11 +68,91 @@ public void setBeaconId(UUID beaconId) {
     this.beaconId = beaconId;
   }
 
-  public UUID getPersonId() {
-    return personId;
+  public String getFullName() {
+    return fullName;
+  }
+
+  public void setFullName(String name) {
+    this.fullName = name;
+  }
+
+  public String getTelephone() {
+    return telephone;
+  }
+
+  public void setTelephone(String telephone) {
+    this.telephone = telephone;
+  }
+
+  public String getEmail() {
+    return email;
+  }
+
+  public void setEmail(String email) {
+    this.email = email;
+  }
```

Comment:
What do you think about using [Lombok](https://projectlombok.org/features/GetterSetter) or similar to reduce the getter/setter boilerplate? (Not used it in anger myself, it's recommended by [Spring in Action](https://www.manning.com/books/spring-in-action-sixth-edition))

Reply:
@zackads I was actually debating that when doing the BeaconUse table as the majority of the file is getters/setters. 

Happy to include it in a separate PR after this? It would certainly reduce boiler plate but would require more setup to get your IDE configured to work with it, otherwise your IDE can't infer getter/setters on classes. 

I'll update the README to capture additional setup required